libs/ui-lib/lib/common/components/ui/formik/RichInputField.tsx

```diff
@@ -104,64 +104,57 @@ const RichInputField: React.FC<RichInputFieldPropsProps> = React.forwardRef(
         id={`form-control__${fieldId}`}
         fieldId={fieldId}
         label={label}
-        helperText={
-          typeof helperText === 'string' ? (
-            helperText
-          ) : (
-            <HelperText fieldId={fieldId}>{helperText}</HelperText>
-          )
-        }
-        validated={isValid ? 'default' : 'error'}
         isRequired={isRequired}
         labelIcon={labelIcon}
       >
         <InputGroup
           className={classNames('rich-input__group', { 'rich_input__group--invalid': !isValid })}
         >
-          <TextInput
-            {...field}
-            {...props}
-            ref={ref}
-            id={fieldId}
-            isRequired={isRequired}
-            aria-describedby={`${fieldId}-helper`}
-            onChange={(value, event) => {
-              !popoverOpen && setPopoverOpen(true);
-              !noDefaultOnChange && field.onChange(event);
-              setTouched(true, false);
-              onChange && onChange(event);
-            }}
-            className="rich-input__text"
-            onBlur={() => {
-              setTouched(true, false);
-              setPopoverOpen(false);
-            }}
-          />
-          <Popover
-            isVisible={popoverOpen}
-            shouldClose={() => setPopoverOpen(false)}
-            shouldOpen={() => setPopoverOpen(true)}
-            aria-label="validation popover"
-            position={PopoverPosition.top}
-            bodyContent={
-              <RichValidation
-                value={value}
-                error={error}
-                richValidationMessages={richValidationMessages as Record<string, string>}
-              />
-            }
-          >
-            <Button variant="plain" aria-label="Validation">
-              {!isValid ? (
-                <ExclamationCircleIcon color={dangerColor.value} />
-              ) : value ? (
-                <CheckCircleIcon color={okColor.value} />
-              ) : (
-                <InfoCircleIcon color={blueInfoColor.value} />
-              )}
-            </Button>
-          </Popover>
+          <InputGroupItem isFill>
+            <TextInput
```

Comment:
Just to confirm, the previous actions from `onChange` are not needed anymore?
(specially the `setPopoverOpen(true)` and `setTouched(true, false)` )

Reply:
Good catch. I've reverted my changes.